examples/transforms/global_vs_local_translation.rs

```diff
@@ -0,0 +1,235 @@
+//! Illustrates the difference between direction of a translation in respect to local object or
+//! global object Transform.
```

Comment:
Can you add this example to the `Cargo.toml` for bevy so that CI will run it?

Reply:
Hmm, I thought this file was deleted. Got confused here. I'll remove it simply.